modules/special-rules/nav.adoc

```diff
@@ -18,43 +18,60 @@
   *** xref:vortex-missile.adoc[]
 
  ** xref:special-abilities.adoc[]
-  *** xref:anti-personnel-ability.adoc[]
-  *** xref:artillery.adoc[]
+
+ ** Special abilities: unit design modifiers
+  *** xref:armour-1.adoc[]
   *** xref:assault.adoc[]
-  *** xref:bridge.adoc[]
-  *** xref:brood-telepathy.adoc[]
-  *** xref:cavalry.adoc[]
-  *** xref:close-support.adoc[]
-  *** xref:deathwing.adoc[]
-  *** xref:engineers.adoc[]
+  *** xref:assault-1.adoc[]
   *** xref:fast.adoc[]
-  *** xref:flak.adoc[]
+  *** xref:firepower-1.adoc[]
   *** xref:heavy-weapons.adoc[]
   *** xref:hero.adoc[]
-  *** xref:hit-and-run.adoc[]
+  *** xref:long-range.adoc[]
+  *** xref:self-propelled-artillery.adoc[]
+  *** xref:shootier.adoc[]
+  *** xref:siege.adoc[]
+
+ ** Special abilities: unit abilities
+  *** xref:artillery.adoc[]
+  *** xref:cavalry.adoc[]
+  *** xref:close-support.adoc[]
+  *** xref:flak.adoc[]
+  *** xref:hitch-x.adoc[] *NEW* equivalent of Transport (X)
+  *** xref:holo-fields.adoc[]
+  *** xref:ignore-terrain.adoc[]
   *** xref:infiltrators.adoc[]
   *** xref:jump-packs.adoc[]
   *** xref:kroot.adoc[]
-  *** xref:long-range.adoc[]
-  *** xref:medic.adoc[]
-  *** xref:open-top.adoc[]
-  *** xref:ordo-malleus.adoc[]
   *** xref:psyker.adoc[]
   *** xref:rampage.adoc[]
   *** xref:rapid-fire.adoc[]
-  *** xref:ravenwing.adoc[]
-  *** xref:recovery.adoc[]
   *** xref:save.adoc[]
-  *** xref:self-propelled-artillery.adoc[]
-  *** xref:shootier.adoc[]
-  *** xref:siege.adoc[]
   *** xref:skimmer.adoc[]
-  *** xref:space-wolves.adoc[]
-  *** xref:stubborn.adoc[]
   *** xref:supreme-commander.adoc[]
-  *** xref:tank-destroyer.adoc[]
+  *** xref:synapse-creature.adoc[]
   *** xref:transport-x.adoc[]
   *** xref:tunneller.adoc[]
+  *** xref:war-engine-supreme-commander.adoc[] *NEW* this is just the ability to re-roll when supporting, for Dominatrixes and maybe add to Gargant Warlords
+
+ ** Special abilities: detachment abilities
+  *** xref:a-dying-race.adoc[]
+  *** xref:brood-telepathy.adoc[]
+  *** xref:dodge.adoc[]
+  *** xref:dauntless.adoc[]
+  *** xref:hit-and-run.adoc[]
+  *** xref:no-retreat.adoc[]
+  *** xref:psychic-beacon.adoc[]
+  *** xref:ravenwing.adoc[]
```

Comment:
Ah, I added Ravenwing earlier, and put some redirects in that and the other similar topics. My thinking was that if we leave these in then, with the 'See [new topic]' redirect, this will help to ease people in as they adjust from Epic 40,000 rules and/or army lists.
We could go either way, but I lean in favour of this 'signposting', at least for now — and it might help to moderate the pushback from some in the community. What say you?

Reply:
Also, I've moved all the 'experimental' topics to the new module, so will need to update this nav file to reflect this — but we can do that at the same time as some other tweaks.

Reply:
> Ah, I added Ravenwing earlier, and put some redirects in that and the other similar topics. My thinking was that if we leave these in then, with the 'See [new topic]' redirect, this will help to ease people in as they adjust from Epic 40,000 rules and/or army lists.
> We could go either way, but I lean in favour of this 'signposting', at least for now — and it might help to moderate the pushback from some in the community. What say you?

I see what you mean, but I don't think they're equivalent to the Anti-tank/Macro-Weapon changes as they were detachment specific rules that only applied to a small subset of lists, rather than being one of the most common special rules in the game.

Reply:
So the principle is the same, but the benefit is less (so perhaps the small gain doesn't justify the extra clutter from redundant stuff)? Yes, I see your point, and I don't like any more clutter than is necessary. Plus, the new topics have 'difference' notes that reference the original, so at least there is a connection somewhere.

It'll just mean that someone looking at an original E40k army list will have to know ER at least a little to map the likes of Ravenwing to Dodge — but then if they're using ER they should want to use our new army lists (and your detachment builder) anyway.

So, I'm happy to go with your suggestion to delete Ravenwing/Deathwing/Space Wolves topics and nav entries.

Reply:
Ravenwing deletion added to commit batch.